source/assets/stylesheets/ie8_fixes.scss

```diff
@@ -31,12 +33,23 @@
 }
 
 // Place icons in the middle of its container
-.footer-social_icons__icon {
+.social_icons__icon {
   padding-top: 6px;
   height: 22px;
 }
 
 // To support full coloured div
-.footer-social-icons {
+.social-icons {
   display: block;
 }
+
+// To set a blog article width
+.blog-header__article-header,
+.blog-article__container,
+.blog-author__container,
+.blog-comments__container,
+.blog-related__container,
+.blog-related__header {
```

Comment:
Thanks Tom. They were set for IE8 because of media queries, so only on IE8 the container took the whole space of the screen. But I agree it's a bit confusing so will try to change it.

Re previous comment I will check with that with Tyler (I was following the blog post design which seems to be narrower then a project story's container)